src/cdf.rs

```diff
@@ -28,10 +28,11 @@ use strum::EnumCount;
 #[derive(Clone, Default)]
 #[repr(C)]
 pub struct CdfContext {
-    pub m: CdfModeContext,
-    pub kfym: Align32<[[[u16; N_INTRA_PRED_MODES + 3]; 5]; 5]>,
     pub coef: CdfCoefContext,
+    pub m: CdfModeContext,
+    pub mi: CdfModeInterContext,
```

Comment:
Where is this field coming from?  I don't see it in the C changes.

Reply:
I basically split `struct CdfModeContext` into two structures such as to make it easier to copy them independently. The C code uses `offsetof()` to copy parts of `struct CdfModeContext`

Reply:
Ah, okay, that makes sense.